Waste.API/Controllers/WasteController.cs

```diff
@@ -68,18 +68,18 @@ public async Task<IActionResult> SaveJourneyWasteType(int? journeyId, int? waste
         }
         
         [HttpPost]
-        [Route("Journey/{journeyId}/WasteType/{selectedWasteType}")]
-        public async Task<ActionResult> SaveWasteType(int? journeyId, string? selectedWasteType)
+        [Route("Journey/{journeyId}/WasteType/{whatHaveYouDoneWaste}")]
```

Comment:
WasteType is the wrong route for a restful API

Reply:
Done.